mmv1/products/compute/terraform.yaml

```diff
@@ -979,6 +979,17 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           - "port_range"
           - "target"
           - "ip_address"
+      - !ruby/object:Provider::Terraform::Examples
```

Comment:
This YAML should stay as it creates documentation and generates acceptance test code as expected 👍